### ISSUE TITLE
Staging: deal with small amount roundings regarding blocking a funder's actions

### ISSUE THREAD

**Issue by ArcAnya** [OpenQDev/OpenQ-Frontend#1535]
Probably low priority since in the end I'm guessing funders could simply add a minimally superior amount to unblock the functionalities - but it could be annoying for the funder if they cannot proceed as planned due to rounding errors.
Annoyance could be also increased if #1427 happens to the user (i.e. many decimals because of JavaScript imprecision on certain nbs)

It's possibly because the $ value is only in the 0.00 decimals place - so when the precision goes even one decimal outside of this, the contract shows insolvent while the contract is actually solvent - blocking all further actions of the funder.

Maybe the solvent / insolvent (and consequential blocking of winner selection etc) should be as precise as possible relying on the full token decimals rather than the rounded $ value ? When weird roundings ocurr as mentioned - we should think of some easy way to deal with them:

Just fyi - same as in #1427 just happened to me again here - and illustrates how small roundings could end up being annoying if not handled properly (I entered 0.07 and JavaScript messed it up again):

![image](https://user-images.githubusercontent.com/75732239/222703045-51c718f2-6dd1-480c-bb7c-c32510f4a483.png)
